contracts/modules/licensing/PILicenseTemplate.sol

```diff
@@ -463,6 +463,7 @@ contract PILicenseTemplate is
         for (uint256 i = 1; i < licenseTermsIds.length; i++) {
             if ($.licenseTerms[licenseTermsIds[i]].commercialUse != commercial) return false;
             if ($.licenseTerms[licenseTermsIds[i]].derivativesReciprocal != derivativesReciprocal) return false;
+            if (derivativesReciprocal && licenseTermsIds[0] != licenseTermsIds[i]) return false;
```

Comment:
Does `derivativesReciprocal` mean *all* terms in the License Terms Config must be strictly equal?